### PR TITLE
update stylo bindings

### PR DESCRIPTION
**Pull request by heycam**
Update stylo bindings for the latest mozilla-central sync.

r? @Manishearth

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14457)

### CODE REVIEW

File: components/style/binding_tools/regen.py [hunk 1]
```diff
@@ -291,7 +292,6 @@
             "nsIURI",
             "nsMainThreadPtrHolder",
             "nsRestyleHint",
-            "nsString",
```
Comment (Manishearth) [inline components/style/binding_tools/regen.py]: why remove this?

Comment (heycam) [inline components/style/binding_tools/regen.py]: I got warnings about it not being used, after my first try doing a regen, so I thought I should!

Comment (Manishearth) [inline components/style/binding_tools/regen.py]: Oh, we import nsAstring anyway.

r=me

**Pull request merged**